zuul.d/junipernetworks-junos-jobs.yaml

```diff
@@ -79,180 +79,59 @@
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python27-stable211
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python27-stable29
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-python35
-    parent: ansible-test-network-integration-junos-vsrx
-    nodeset: vsrx3-18.4R1-python35
-    vars:
-      ansible_test_python: 3.5
-
 - job:
     name: ansible-test-network-integration-junos-vsrx-python36
     parent: ansible-test-network-integration-junos-vsrx
     nodeset: vsrx3-18.4R1-python36
     vars:
       ansible_test_python: 3.6
 
+# ====================================================================================
 - job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36-stable211
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 3.6
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36-stable29
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-junos
+    parent: ansible-network-junos-appliance
 
 - job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python36
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-junos
+    parent: network-ee-integration-tests
+    nodeset: vsrx3-18.4R1-python38
 
 - job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-libssh-python36
-    parent: ansible-test-network-integration-junos-vsrx-network_cli-python36
+    name: network-ee-integration-tests-junos-libssh
```

Comment:
we should name these network-ee-integration-tests-junos-vsrx, given we have a 2nd nodeset for junos